bin/get_twinkles_visits.py

```diff
@@ -4,20 +4,57 @@
 file given a fieldID.
 """
 from __future__ import absolute_import
+import os
 import argparse
+import pandas as pd
 from desc.twinkles import get_twinkles_visits
+from desc.twinkles import OpSimOrdering
+from lsst.utils import getPackageDir
 
 parser = argparse.ArgumentParser(description="Write an ascii file of visits derived from an OpSim db file given a fieldID")
 parser.add_argument('opsimDB', help='OpSim database sqlite file')
 parser.add_argument('--fieldID', type=int, default=1427,
                     help='ID number of the desired field')
 parser.add_argument('--outfile', type=str, default=None, help='output file')
+parser.add_argument('--orderObsHistIDsByDesign', type=bool, default=True,
+                    help='order obsHistIDs as decided for Twinkles 3')
+parser.add_argument('--randomForestPickleFileName', type=str, default='RF_pickle.p',
```

Comment:
So I actually have two suggestion here, neither of which are compulsory:
* Just have a single option for the random forest file location, i.e., `--RF_pickle_file`.  This could be set to `os.path.join(getPackageDir('Twinkles'), 'data', 'RF_pickle.p')` if the default is `None`.
* make the option names shorter and easier to type.